server/services/project_service.go

```
@@ -0,0 +1,319 @@
+			`docker run -d -l "traefik.http.routers.%s.rule=Host(\"%s\")" -l "traefik.http.services.%s.loadbalancer.server.port=8000" --name %s %s`,
+			project.Name, hostName, project.Name, project.Name, project.Name,
+		),
+		OutputCallback: func(s string) {
+			fmt.Println("> ", s)
+			network := models.Network{
+				ProjectId: project.Id,
+				ProcessId: s,
+				HostName:  hostName,
+			}
+			err := p.db.Create(&network).Error
+			if errors.Is(gorm.ErrDuplicatedKey, err) {
```

golangci-lint (1.62.2)

SA1032: arguments have the wrong order

(staticcheck)